### ISSUE TITLE
user_cas available for ownCloud 8?

### ISSUE THREAD

**Issue by titansmc** [owncloud/user_cas#11]

**Comment by jgribonvald** [comment]
this is working for us with owncloud 8.4

**Comment by titansmc** [comment]
Thanks a lot, are you combining CAS + LDAP?

**Comment by jgribonvald** [comment]
yes, but it seems there are some bugs, user_cas is recreating a user account overriding ldap account.

**Comment by jgribonvald** [comment]
user_cas registering user attributes should be disable by a property as example

**Comment by timm2k** [comment]
Since pull request #15 has been applied user_cas works for me with OC 8.1.1 and LDAP.

**Comment by tinylcy** [comment]
Hello everyone!
At first please forgive my poor English, I'm trying to bind ownCloud with CAS+LDAP,  and I'm trying to achieve this by user_cas, but I'm really new to these things, so does anyone can recommend me some documents or advice or blogs? Thx very much.
